lld/ELF/Arch/LoongArch.cpp

```diff
@@ -971,6 +1017,7 @@ void LoongArch::finalizeRelax(int passes) const {
           switch (newType) {
           case R_LARCH_RELAX:
             break;
+          case R_LARCH_B26:
```

Comment:
#123576 does not change `r.expr`. Which one is appropriate?

Reply:
Ah, it should remain `R_PLT_PC`. I'll see if I should just close this instead of #123576 because I didn't know that PR even existed until today morning.